superx/information_extractors/branch_info_extractor.py

```diff
@@ -1,11 +1,17 @@
 import gzip
 from bs4 import BeautifulSoup
 import xml.etree.ElementTree as et
-from app import supermarket_info_dictionary, session, db
-from models import Branch
+import os
+import sys
 import logging
 import requests
 
+add_to_python_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
+sys.path.append(add_to_python_path)
```

Comment:
the ci runs on ubunto and does not recognize the path to app and models. This line adds the path to the system. same as item_info extractor

Reply:
that should be a comment in the code, not the PR